src/renderer/store/sharedHelpers.ts

```diff
@@ -33,6 +33,8 @@ export interface ApplicationStore {
   collab: CollabClientSessionState | CollabClientInitialState | null;
   // Op queue session state for pending actions when in a collab session
   opQueue: OpQueueItem[];
+  isVideoPaused: boolean;
```

Comment:
Hi Chloe, I'm not really too sure what you mean by putting the 3 states in an interface so they can be handled by a single reducer. Don't I need a separate reducer for each state? Otherwise, how would I use useEffect() to only activate on the change of one state.

Reply:
You don’t need a separate reducer for each. When it comes to useEffect, you can get the variable of interest from useSelector and go as fine grained as you want.

Reply:
Is there any templates in the codebase on how I should implement the interface? I had a look at selection but the actions/reducer is really confusing.

Reply:
If you’ve pulled from latest master selection is an okay example - although it does have the self/others thing. Perhaps a better example is the undoStack reducer - aside from being a bit meta it shows the benefit of updating multiple fields at once within the reducer.

Let me know if you want to pair on this

Reply:
Hey Chloe, just pushed a bunch of commits which implements skipping forward/backwards with left and right arrow keys. State seems to be updating correctly and the video preview is able to skip 10 seconds forward/backwards regardless of whether the video is paused or playing. I'm just not sure if the interface/reducer is in the structure you wanted so it'd be good to double check before I finish up this pr.